roles/tas_single_node/defaults/main.yml

```diff
@@ -121,6 +121,11 @@ tas_single_node_oidc_issuers:
     url: https://keycloak-keycloak-system.apps.platform-sts.pcbk.p1.openshiftapps.com/auth/realms/trusted-artifact-signer
     type: email
 
+tas_single_node_meta_issuers:
```

Comment:
I think we shouldn't add an empty entry, because that might actually be an invalid configuration value. I think we should just use `tas_single_node_meta_issuers: []` - just an empty list to be explicit that there are no meta issuers by default.

Reply:
done :)